libs/martech/analytics.js

```diff
@@ -1,9 +1,16 @@
-export function processTrackingLabels(text, charLimit = 20) {
-  return text?.trim().replace(/\s+/g, ' ').split('|').join(' ')
+export function processTrackingLabels(text, config, charLimit = 20) {
+  if (!config) {
```

Comment:
What exactly is the purpose of the `config` argument and its re-assignment, since it doesn't seem to be used to generate the `analyticsValue`?

Reply:
That's in preparation for the feature to preserve the analytics in English during localization.  I've already started work on it and I know I'm going to need the config in this function.